launcher/minecraft/mod/ResourcePackFolderModel.cpp

```diff
@@ -102,6 +104,9 @@ QVariant ResourcePackFolderModel::data(const QModelIndex& index, int role) const
             return {};
         }
         case Qt::ToolTipRole: {
+            if (column == ActiveColumn && !at(row).validMCMeta()) {
```

Comment:
IMO it'd be better to have this on the name column (the same as the symlink warning) and have the messages stack